### PR TITLE
docs: add an example of deploying authelia lite on docker swarm

### PR DESCRIPTION
**Pull request by wuhanstudio**
The authelia lite has been successfully deployed to my docker swarm after adding some modifications to: https://github.com/authelia/authelia/pull/1112

http://traefik.wuhanstudio.cc/
http://public.wuhanstudio.cc/
http://secure.wuhanstudio.cc/

### CODE REVIEW

**Comment by james-d-elliott** [comment]
Thanks for choosing to contribute! Did you test this? Specifically the fact that the X-Forwarded-For header is correct. Previous versions of Traefik required to be scheduled on the swarm master node. The logs should clearly show if there is an issue - basically they should not have the users actual IP.

**Comment by wuhanstudio** [comment]
> Thanks for choosing to contribute! Did you test this? Specifically the fact that the X-Forwarded-For header is correct. Previous versions of Traefik required to be scheduled on the swarm master node. The logs should clearly show if there is an issue - basically they should not have the users actual IP.
> 
> See these docs: https://doc.traefik.io/traefik/providers/docker/#docker-api-access_1

Sorry, the placement constraints was removed by accident.

I have tested out on two of my clusters:

http://public.wuhanstudio.uk/
http://traefik.wuhanstudio.uk/
http://private.wuhanstudio.uk/

http://public.wuhanstudio.cc/
http://traefik.wuhanstudio.cc/
http://private.wuhanstudio.cc/

**Comment by wuhanstudio** [comment]
> Thanks for choosing to contribute! Did you test this? Specifically the fact that the X-Forwarded-For header is correct. Previous versions of Traefik required to be scheduled on the swarm master node. The logs should clearly show if there is an issue - basically they should not have the users actual IP.
> 
> See these docs: https://doc.traefik.io/traefik/providers/docker/#docker-api-access_1

Sorry, the placement constraints was removed by accident.

I have tested out on two of my clusters:

http://public.wuhanstudio.uk/
http://traefik.wuhanstudio.uk/
http://secure.wuhanstudio.uk/

http://public.wuhanstudio.cc/
http://traefik.wuhanstudio.cc/
http://secure.wuhanstudio.cc/

**Comment by james-d-elliott** [comment]
> 
> 
> > Thanks for choosing to contribute! Did you test this? Specifically the fact that the X-Forwarded-For header is correct. Previous versions of Traefik required to be scheduled on the swarm master node. The logs should clearly show if there is an issue - basically they should not have the users actual IP.
> > See these docs: https://doc.traefik.io/traefik/providers/docker/#docker-api-access_1
> 
> Sorry, the placement constraints was removed by accident.
> 
> I have tested out on two of my clusters:
> 
> http://public.wuhanstudio.uk/
> http://traefik.wuhanstudio.uk/
> http://secure.wuhanstudio.uk/
> 
> http://public.wuhanstudio.cc/
> http://traefik.wuhanstudio.cc/
> http://secure.wuhanstudio.cc/
> 
> Besides, I created a github repo that contains full configuration files:
> 
> https://github.com/wuhanstudio/authelia-docker-swarm

No drama at all. I wasn't sure it was still needed, the docs are not entirely clear on that.

File: docs/community/example-of-authelia-lite-on-docker-swarm.md [hunk 1]
```diff
@@ -0,0 +1,182 @@
+---
+layout: default
+title: Example of authelia lite on docker swarm
+parent: Community
+nav_order: 3
```
Comment (james-d-elliott) [inline docs/community/example-of-authelia-lite-on-docker-swarm.md:5]:
```suggestion
nav_order: 4
```

**Review by james-d-elliott — APPROVED** [review]

**Review by james-d-elliott — APPROVED** [review]
LGTM! Thanks for your contribution. @all-contributors please add @wuhanstudio for docs.

**Comment by james-d-elliott** [comment]
Guess the bot didn't see that message.

@all-contributors please add @wuhanstudio for docs

**Pull request merged**